alpha/lib/enums/EntryServerNodeType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * @package Core
+ * @subpackage model.enum
+ */
+interface EntryServerNodeType extends BaseEnum
```

Comment:
Here as well this is the same as MediaServerIndex so there is no reason to keep both ... choose one or the other and create the necessary client libs accordingly.